lua/autorun/client/cfc_attention_monitoring_cl_init.lua

```diff
@@ -61,33 +68,33 @@ local function formatAfkTime( rawTime )
 end
 
 local function drawIcon( ply )
-    if not IsValid( ply ) then
+    if not ply or ply == NULL then
```

Comment:
Can you localize `NULL`? I forget if that works.

Reply:
i am actually not sure aswell xP